operations/rack_resiliency/rack_resiliency.md

```diff
@@ -0,0 +1,490 @@
+
+`Kubernetes` zone prefix updation: Update the `spec.services.k8s_zone_prefix` section in the `customizations.yaml` file with the required `Kubernetes` zone prefix. This `customizations.yaml` file will be present in the root directory of the CSM tarball.
+
+`Ceph` zone prefix updation: Update the `spec.services.Ceph_zone_prefix` section in the `customizations.yaml` file with the required `Ceph` zone prefix. This `customizations.yaml` file will be present in the root directory of the CSM tarball.  
+
+During the CSM installation or upgrade, a `Kubernetes` secret named `site-init` under `loftsman` will be created with the prefixes updated in the fields `spec.services.k8s_zone_prefix` and `spec.services.ceph_zone_prefix`. Further, the same secret will be checked for zone label prefixes during the `Kubernetes` and `Ceph` zone creation for Master, Worker and Storage nodes. If the prefixes is defined, then the zones will be created in the format of `k8s_zone_prefix + rack_id` and `ceph_zone_prefix + rack_id`.
+
+**Examples:**
+
+If the `spec.services.k8s_zone_prefix` has a value of `test-system` and the rack-id is `x3000`, then the `Kubernetes` zones will be created with the labels of value `test-system-x3000`
+
+If the `spec.services.ceph_zone_prefix` has a value of `test-storage-system` and the rack-id is `x3000`, then the `Ceph` zones will be created with the labels of value `test-storage-system-x3000`
+
+If the `spec.services.k8s_zone_prefix` has no value defined and the rack-id is `x3000`, then the `Kubernetes` zones will be created with the labels of value `x3000`
+
+If the `spec.services.ceph_zone_prefix` has no value defined and the rack-id is `x3000`, then the `Ceph` zones will be created with the labels of value `x3000`
+   
+#### 3 Auto deploy of RR CFS ansible plays with bootprep
+
+RR CFS `ansible` plays are deployed as part of IUF bootprep stage using the [Install and Upgrade Framework (IUF)](../../glossary.md#install-and-upgrade-framework-iuf).
+It is initiated during bootprep (`management-nodes-rollout`) in order to enable/configure RR with placement discovery, placement validation, zoning and applying `kyverno` policy automatically during boot time.
+
+If placement validation criteria is met as explained in the solution overview section "Validate distribution of management/ managed nodes", it will configure RR zoning and apply `kyverno` policy otherwise it will exit the RR configuration with respective placement validation failure message.
+
+#### 4 Auto deploy of RRS (Rack Resiliency Service) helm chart
+
+RRS helm chart consisting RRS pod with RRS API (CLI backend) and RMS (Resiliency Monitoring Service) containers will be deployed post `management-nodes-rollout` IUF step.
+
+Post `management-nodes-rollout`, as part of post hook, it will check if the Rack Resiliency is enabled and zones (`Kubernetes`/ `Ceph`) are created. If present, it will then deploy the RRS chart otherwise skip to deploy.

```

Comment:
So how does this work in the fresh install case? IUF isn't involved, so there will be no IUF post hooks involved. I assume the chart is going to be deployed alongside all of the others. I don't see anything in the platform manifest file that would exclude it.

Reply:
This has moved to `operations/rack_resiliency/enable_setup.md`, but it is still an issue there